test/IISExpress.FunctionalTests/WindowsAuthTests.cs

```diff
@@ -26,13 +26,14 @@ public static TestMatrix TestVariants
             => TestMatrix.ForServers(DeployerSelector.ServerType)
                 .WithTfms(Tfm.NetCoreApp22, Tfm.Net461)
                 .WithAllApplicationTypes()
-                .WithAllAncmVersions();
+                .WithAllAncmVersions()
+                .WithAllHostingModels();
 
         [ConditionalTheory]
         [MemberData(nameof(TestVariants))]
         public async Task WindowsAuthTest(TestVariant variant)
         {
-            var deploymentParameters = _fixture.GetBaseDeploymentParameters(variant);
+            var deploymentParameters = _fixture.GetBaseDeploymentParameters(variant, publish: true);
```

Comment:
One of the inproc tests was failing without setting publish to true.

Reply:
Why?